BUILD

```diff
@@ -0,0 +1,25 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@rules_rust//rust:defs.bzl", "rust_clippy", "rust_library", "rustfmt_test")
+
+rust_library(
+    name = "sbi-rs",
+    srcs = glob(["src/**/*.rs"]),
+    crate_root = "src/sbi.rs",
+    proc_macro_deps = ["@salus-index//:enum_dispatch"],
+    deps = [
+        "@salus-index//:arrayvec",
```

Comment:
This is awkward, and makes it more difficult to import this repo with a different crate universe.

The "right" way to do this is probably to set up a crate universe for sbi-rs and set up all of the code to populate the universe, and invoke it in Salus (and anywhere else we need it -- I'm beginning to integrate sbi-rs into my own project). That's a hassle and seems like overkill for a tiny library crate like this.